auv_sim/auv_sim_description/auv_sim_description/models/robosub_bin/ball_two.sdf

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
```

Comment:
you can spawn 2 object under different names in gazebo, why need the duplicate ball_two ? remove this